CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 ## New Features
+
```

Comment:
can we have a changelog entry for the improvement? :)

Reply:
It doesn't really fit into any of the existing categories... should we open a new one for: `Improvements` or `Performance`?

Reply:
I just threw the statsd one into the new features but yes it's not perfect, Improvements, works Performance works too, Fixes would work (if we consider low performance a bug but it really isn't fitting perfect either), new feature works (if we consider fast to be a feature, but not really either :P:)